src/styles/blocks/header.scss

```diff
@@ -0,0 +1,76 @@
+.header {
+  background-color: $background-color-special-white;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-sizing: border-box;
+  width: 100%;
+
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  &__nav {
+    font-size: 12px;
+    font-weight: 500;
+
+    overflow-x: auto;
+    white-space: nowrap;
+  }
+
+  &__nav-list {
+    height: 60px;
+    line-height: 60px;
+    margin: 0;
+    padding: 0;
+    text-align: center;
+    font-size: 0;
+  }
+
+  &__nav-item {
+    margin: 0 10px;
+    text-decoration: none;
+    display: inline-block;
+  }
+
+  &__nav-item:first-child {
+    margin-left: 0;
+  }
+
+  &__nav-item:last-child {
+    margin-right: 0;
+  }
+
+  &__logo-link {
+    height: 40px;
+    width: 40px;
+    margin-right: 20px;
+  }
+
+  &__nav-link {
+    position: relative;
+    display: block;
+    color: #060b35;
+    font-size: 12px;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    &:hover {
+      color: $button-color;
```

Comment:
Ensure that the SCSS variable `$button-color` is defined in your stylesheets. If not, you need to define it or replace it with a valid color value.